src/css/hero.css

```diff
@@ -1,16 +1,178 @@
 /* hero styles */
 .hero-section .container {
+  width: 280px;
```

Comment:
please don't use width for containers. min-width should be here.
Next - 
Please don't override the 'container' class, as it's used throughout the site. It's simpler to just add another class, such as 'hero-container', and append the styles you need that aren't in the 'container' class. Or, if there are any discrepancies, just opt not to use it.